tests/setup.js

```diff
@@ -0,0 +1,73 @@
+import 'babel-polyfill';
+import { assert } from 'chai';
+import sinon from 'sinon';
+import areIntlLocalesSupported from 'intl-locales-supported';
+
+class LocalStorageMock {
+  constructor() {
+    this.store = {};
+  }
+  clear() {
+    Object.keys(this.store).forEach((key) => {
+      delete this.store[key];
+    });
+  }
+  getItem(key) {
+    return this.store[key] || null;
+  }
+  setItem(key, value) {
+    this.store[key] = value;
+  }
+  removeItem(key) {
+    delete this.store[key];
+  }
+  key(index) {
+    const keys = Object.keys(this.store);
+    return keys[index] || null;
+  }
+  get length() {
+    return Object.keys(this.store).length;
+  }
+}
+global.localStorage = new LocalStorageMock();
+
+const localesMyAppSupports = [
+  'de-DE', 'fr',
+];
+
+if (global.Intl) {
+  // Determine if the built-in `Intl` has the locale data we need.
+  if (!areIntlLocalesSupported(localesMyAppSupports)) {
+    // `Intl` exists, but it doesn't have the data we need, so load the
+    // polyfill and patch the constructors we need with the polyfill's.
+    // eslint-disable-next-line global-require
+    const IntlPolyfill = require('intl');
+    Intl.NumberFormat = IntlPolyfill.NumberFormat;
+    Intl.DateTimeFormat = IntlPolyfill.DateTimeFormat;
+  }
+} else {
+  // No `Intl`, so use and load the polyfill.
+  // eslint-disable-next-line global-require
+  global.Intl = require('intl');
+}
+
+
+const realSinon = sinon;
+global.before = beforeAll;
```

Comment:
I think it would be better to update all of this code, i.e. just rename `before()` to `beforeAll()`.